lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -1109,15 +1109,8 @@ namespace Js
 
         if (callInfo.Count < 2)
         {
-            if (pNew == nullptr)
```

Comment:
>if (pNew == nullptr) [](start = 12, length = 20)

what happens to the != nullptr cases ? You recreate new array everytime ? When will pNew is not nullptr ?